gobrew.go

```diff
@@ -266,9 +268,24 @@ func (gb *GoBrew) ListRemoteVersions(print bool) map[string][]string {
 
 // CurrentVersion get current version from symb link
 func (gb *GoBrew) CurrentVersion() string {
-	fp, err := filepath.EvalSymlinks(gb.currentBinDir)
-	if err != nil {
-		return "None"
+	// https://github.com/golang/go/issues/63703
+	var fp string
+	if runtime.GOOS == "windows" {
```

Comment:
We have a file, `gobrew_windows.go`, which houses the code used only in windows.

It seems that you need to create a separate function like `evalSymlinks`, for which `gobrew_unix.go` will have a normal implementation, and in `gobrew_windows.go` will be specific to windows.

And the code will be easier to read.